boot/zephyr/VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 1
+VERSION_MINOR = 11
```

Comment:
We are still at minor 10. Why patchlevel is 99?

Reply:
Currently at 1.11.0-dev according to a97f009d209deb20278a7e12234364bad02533f6, put it to 99 to copy the zephyr style release pattern of "not a proper release" since the "dev" text in the VERSION file is completely ignored